engine/execution/ingestion/engine_test.go

```diff
@@ -66,6 +66,8 @@ type testingContext struct {
 	identity            *flow.Identity
 	broadcastedReceipts map[flow.Identifier]*flow.ExecutionReceipt
 	collectionRequester *module.MockRequester
+
+	mu sync.Mutex
```

Comment:
Go Vet is mad because adding this as a value rather than a pointer means when it gets passed around the lock gets copied

I believe what you're intending to do would still work with this as a pointer, we'd just have to make sure to initialize the lock